internal/cli/atlas/deployments/options/deployment_opts_pre_run.go

```diff
@@ -108,11 +107,10 @@ func (opts *DeploymentOpts) AtlasDeployments(projectID string) ([]Deployment, er
 		ItemsPerPage: MaxItemsPerPage,
 	}
 
-	projectClusters, err := opts.AtlasClusterListStore.ProjectClusters(projectID, listOpts)
+	atlasClusters, err := opts.AtlasClusterListStore.ProjectClusters(projectID, listOpts)
 	if err != nil {
 		return nil, err
 	}
-	atlasClusters := projectClusters.(*admin.PaginatedAdvancedClusterDescription)
```

Comment:
on line 119, 120, 126 and 127, can you update to use the getters instead of pointers?